pkg/webui/lib/download-object-as-file.js

```diff
@@ -0,0 +1,22 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+export default function downloadObjectAsFile(dataStr, filename) {
+  const node = document.createElement('a')
+  node.setAttribute('href', dataStr)
+  node.setAttribute('download', `${filename}.json`)
```

Comment:
```suggestion
  node.setAttribute('download', fiilename)
```

To keep this more flexible.